app/logDirectory.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs')
+const {join} = require('path')
+
+const logPath = join(__dirname, '..', 'log')
```

Comment:
Use `resolve` instead of `join`. `resolve` always result in a relative path, while `join` doesn't. It's doesn't make any different here, it's just a good practice.